### ISSUE TITLE
Upgrade GridPath to work with Pyomo v5.7

### ISSUE THREAD

**Issue by milindsmart** [blue-marble/gridpath#660]
On GridPath v0.2, opening GridPath GUI or running `gridpath_run_scenario` results in the following ImportError

```
ImportError: pyomo.environ failed to import pyomo.opt.plugins:
Original ImportError: pyutilib.enum has been removed.

Python 3 now has an enum implementation in the standard library (also
available for older Python versions as the third-party enum34 PyPI
package) that supersedes this library.

```

**Comment by anamileva** [comment]
Closed with v0.7.0.

**Comment by milindsmart** [comment]
For completeness : https://github.com/Pyomo/pyomo/issues/1525
